app/client/components/GridView.js

```diff
@@ -295,6 +295,15 @@ GridView.gridCommands = {
     }
     this.cursor.rowIndex(this.cursor.rowIndex() - 1);
   },
+  cursorLeft: function() {
+    // This conditional exists so that when users have the cursor in the leftmost column but
+    // are not scrolled to the left i.e. in the case of a wide column, pressing left again will
+    // scroll the pane to the left.
+    if (this.cursor.fieldIndex() === 0) {
+      this.scrollPane.scrollLeft = 0;
+    }
+    this.cursor.fieldIndex(this.cursor.fieldIndex() - 1);
+  },
```

Comment:
Can revert if controversial. (One of the fixes uses this new behavior.)

If ok to include, I can fix the remaining two directions in a separate diff so that the behavior is consistent.

Reply:
This fix makes sense to me!